assets/airports/egcc.json

```diff
@@ -0,0 +1,582 @@
+ {
+  "name": "Manchester Airport",
+  "level": "medium",
+  "radio": {
+    "twr": "Manchester Delivery",
```

Comment:
It's not Manchester Delivery, thanks for reminding me to change that! The charts say "TWR : MANCHESTER DELIVERY" whereas the actual tower is Manchester Tower, which is the landing and takeoff clearance.